### ISSUE TITLE
Handling d-declaration in script and global statements

### ISSUE THREAD

**Issue by jcouv** [dotnet/roslyn#13716]
Should we allow `(int a, int b) = (1, 2);` in script? 
Most likely, yes. `a` and `b` should be fields.

This has implications on the semantic model for d-declarations outside a class or method.

**Comment by tmat** [comment]
Yes, we definitely should. The variables would be indeed fields. @AlekseyTs is looking into similar issues around ```out var```.

**Comment by jcouv** [comment]
Fixed by #13957

**Comment by gafter** [comment]
@jcouv there is still a test skipped with this issue as a skip reason in master.
